docs/running-workflows/supported-systems/cwl/index.md

```diff
@@ -1,3 +1,178 @@
 # CWL
 
 The Common Workflow Language standard originated in life science community.
+
+Latest REANA 0.8 release series uses ``cwltool==3.1.20210628163208``.
+
+Eearlier REANA 0.7 release series used ``cwltool==1.0.20191022103248``.
+
+## CWL v1.0 specification conformance results
+
+REANA 0.8.0 tested on 2022-01-11
+
+- 183 tests passed
+- 14 failures
+
+```console
```

Comment:
Instead of console, let's make it a sublist of 14 failures.

Reply:
We could later think about adding some expand/collapse feature, so that readers would immediately see the overview only (183 passed, 14 fails), and they could click on "expand" to unroll the second sublist with all the failures.  It'll be probably nicer to do it with lists and sublists rather than with console areas.

Reply:
Sublist takes a lot of space, that's why I went for `console` initially, since it renders it in a more compact way.
I've now added a expand/collapse feature:
 - had to move the list of failed tests outside the overview (183 passed, 14 fails) to the bottom due to the way it's being rendered in our docs
 - regular list was not rendered correctly in the expanded panel, so I had to use `plaintext` there
 - had to disable the linter warning to allow `html` tag usage

Let me know what you think about it